homework/test/test-list.js

```diff
@@ -1,8 +1,24 @@
+var MathFunc=function(){};
+
+MathFunc.prototype  = {
```

Comment:
It's better to register this method in the manager, rather than just write it in the test file.